tutorials/2d/custom_drawing_in_2d.rst

```diff
@@ -7,11 +7,9 @@ Why?
 ----
 
 Godot has nodes to draw sprites, polygons, particles, and all sorts of
-stuff. For most cases, this is enough; but not always. Before crying in fear,
-angst, and rage because a node to draw that specific *something* does not exist...
-it would be good to know that it is possible to easily make any 2D node (be it
-:ref:`Control <class_Control>` or :ref:`Node2D <class_Node2D>`
-based) draw custom commands. It is *really* easy to do it, too.
```

Comment:
I guess this was supposed to be light-hearted, but it came off as really patronizing :man_shrugging: